### PR TITLE
Display name again

### PR DESCRIPTION
**Pull request by zlesnr**

### CODE REVIEW

File: internal/install/types/types.go [hunk 1]
```diff
@@ -104,86 +104,90 @@ var OpenInstallationTargetTypeTypes = struct {
...
+	// Friendly name of the integration
+	DisplayName string `json:"displayName,omitempty"`
 	// The full contents of the recipe file (yaml)
 	File string `json:"file"`
 	// The ID
-	ID string `json:"id"`
+	ID string `json:"id,omitempty"`
 	// List of variables to prompt for input from the user
 	InputVars []OpenInstallationRecipeInputVariable `json:"inputVars"`
+	// Go-task's taskfile definiton (see https://taskfile.dev/#/usage)
```
Comment (devfreddy) [inline internal/install/types/types.go:142]: Not important but maybe to satisfy your linter, correct the spelling of `definiton` to `definition`

**Review by devfreddy — DISMISSED** [review]
👍 with a low-priority comment on a spelling thing

File: internal/install/types/types.go (hunk 1 above)
Comment (devfreddy) [reply internal/install/types/types.go:142]: Also, apologies, b/c this almost definitely came from you copy/pasting my spelling mistake 😂

Comment (zlesnr) [reply internal/install/types/types.go]: This code is all generated, so what is stored in the schema is what gets written here.  I'll correct it in place now.

**Review by devfreddy — APPROVED** [review]
:shipit:

**Pull request merged**